cristin-import/src/test/resources/features/GeneralMappingRules.feature

```diff
@@ -8,27 +8,26 @@ Feature: Mappings that hold for all types of Cristin Results
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource has an additional identifier with key "Cristin" and value 12345
 
-  Scenario: Map returns NVA Resource with main title being the Cristin title annotated as
-  Original Title when there is only one CristinTitle and it is annotated as original
+  Scenario: NVA Resource gets as Main Title the single Cristin title which is annotated as
+  "Original Title" (i.e., the Cristin entry has no more titles except for the original title).
```

Comment:
NVA Resource gets the single Cristin title which is annotated as
  "Original Title" (i.e., the Cristin entry has no more titles except for the original title) as Main Title.